dash_auth/public_routes.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import absolute_import
+import inspect
+import os
+
+from dash import Dash, callback
+from dash._callback import GLOBAL_CALLBACK_MAP
+from dash import get_app
+from flask import Flask
+from werkzeug.datastructures import ImmutableDict
+from werkzeug.routing import Map, MapAdapter, Rule
+
+# Add PUBLIC_ROUTES in the default Flask config
+DASH_PUBLIC_ASSETS_EXTENSIONS = "js,css,svg,png,webp,gif"
+BASE_PUBLIC_ROUTES = [
+    f"/assets/<path:path>.{ext}"
+    for ext in os.getenv(
+        "DASH_PUBLIC_ASSETS_EXTENSIONS",
+        DASH_PUBLIC_ASSETS_EXTENSIONS,
+    ).split(",")
+] + [
```

Comment:
@RenaudLN Shouldn't we keep these routes private until at least one 'real' route is made public?